doc/guides/device_mgmt/smp_groups/smp_group_1.rst

```diff
@@ -0,0 +1,401 @@
+    | ``3``  | ``1``        |  ``1``         |
+    +--------+--------------+----------------+
+
+CBOR data of response command:
+
+
+.. code-block:: none
+
+    {
+        (str,opt)"off"  : (uint)
+        (str)"rc"       : (int)
+        (str,opt)"rsn"  : (str)
+    }
+
+where
+
+.. table::
+    :align: center
+
+    +-----------------------+---------------------------------------------------+
+    | "off"                 | offset of last successfully written byte of update|
+    +-----------------------+---------------------------------------------------+
+    | "rc"                  | :ref:`mcumgr_smp_protocol_status_codes`           |
+    +-----------------------+---------------------------------------------------+
+    | "rsn"                 | Optional string that clarifies reason for an      |
+    |                       | error; specifically useful for error code ``1``,  |
+    |                       | unknown error.                                    |
+    +-----------------------+---------------------------------------------------+
+
+The "off" field is only included in responses to successfully processed requests;
+if "rc" is negative the "off' may not appear.
+
+Image erase
+***********
+
+The command is used for erasing image slot on a target device.
+
+.. note::
+    Currently Zephyr version of mcumgr is hardcoded to always erase secondary slot
+    of the first image.
+
+Image erase request
+===================
+
+Image erase request header fields
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``2``  | ``1``        |  ``5``         |
+    +--------+--------------+----------------+
+
+The command sends sends empty CBOR map as data.
+
+Image erase response
+====================
+
+Image erase response header fields
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``3``  | ``1``        |  ``5``         |
+    +--------+--------------+----------------+
+
+CBOR data of response command:
+
+.. code-block:: none
+
+    {
+        (str)"rc"       : (int)
+        (str,opt)"rsn"  : (str)
+    }
+
+where
+
+.. table::
+    :align: center
+
+    +-----------------------+---------------------------------------------------+
+    | "rc"                  | :ref:`mcumgr_smp_protocol_status_codes`           |
+    +-----------------------+---------------------------------------------------+
+    | "rsn"                 | Optional string that clarifies reason for an      |
+    |                       | error; specifically useful for error code ``1``,  |
+    |                       | unknown error.                                    |
+    +-----------------------+---------------------------------------------------+
+
+.. note::
```

Comment:
It may be beneficial to add, that erase command may take long time (~20 seconds), depending on the size of the flash.

Reply:
Will do, btw: https://github.com/zephyrproject-rtos/zephyr/issues/42420

Reply:
Done